treeherder/model/migrations/0009_add_is_try_repo.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-03-09 18:59
```

Comment:
Oh, I just copy/pasted https://github.com/mozilla/treeherder/blob/master/treeherder/model/migrations/0007_add_performance_data_expiry_option.py and changed it to add the new field. Was there some better way to create this?